biggraphite/plugins/carbon.py

```diff
@@ -315,7 +322,7 @@ def _createOneMetric(self):
 
         self.cache.create_metric(metric)
         self.tag(metric_name)
-        CREATES.inc()
+        CREATES_DEQUEUED.inc()
```

Comment:
I think you should move that line 307

Reply:
Yes, I messed up. I will do a corrective PR.